src/coreComponents/dataRepository/LogLevelsInfo.hpp

```diff
@@ -0,0 +1,193 @@
+{
+  static constexpr int getMinLogLevel() { return 1; }
+  static constexpr std::string_view getDescription() { return "Information on global solution scaling factor"; }
+};
+
+struct TimeStep
+{
+  static constexpr int getMinLogLevel() { return 1; }
+  static constexpr std::string_view getDescription() { return "Information on the timestep"; }
+};
+
+struct SolverTimers
+{
+  static constexpr int getMinLogLevel() { return 1; }
+  static constexpr std::string_view getDescription() { return "Information on solver timers"; }
+};
+
+struct ScreenLinearSystem
+{
+  static constexpr int getMinLogLevel() { return 1; }
+  static constexpr std::string_view getDescription() { return "Output to screen the assembled linear system and solutions (matrices and vectors)"; }
+
+};
+
+struct FileLinearSystem
+{
+  static constexpr int getMinLogLevel() { return 2; }
+  static constexpr std::string_view getDescription() { return "Output to file the assembled linear system and solutions (matrices and vectors)"; }
+};
+
+struct SolverConfig
+{
+  static constexpr int getMinLogLevel() { return 1; }
+  static constexpr std::string_view getDescription() { return "On non convergance, Information about testing new configuration and print the time step"; }
+};
+
+struct ResidualNorm
+{
+  static constexpr int getMinLogLevel() { return 1; }
+  static constexpr std::string_view getDescription() { return "Print residual norm"; }
+};
+
+struct ResidualValues
+{
+  static constexpr int getMinLogLevel() { return 1; }
+  static constexpr std::string_view getDescription() { return "Print the residual values"; }
+};
+
+struct LinearSystem
+{
+  static constexpr int getMinLogLevel() { return 1; }
+  static constexpr std::string_view getDescription() { return "Information oon linear system"; }
+};
+
+struct CrossflowWarning
+{
+  static constexpr int getMinLogLevel() { return 1; }
+  static constexpr std::string_view getDescription() { return "If the well is injector and crossflow enabled, display informations about crossflow for injectors"; }
+};
+
+struct HydraulicAperture
+{
+  static constexpr int getMinLogLevel() { return 1; }
+  static constexpr std::string_view getDescription() { return "Information on aperture and hydraulic aperture"; }
+};
+
+struct SolverTimeStep
+{
+  static constexpr int getMinLogLevel() { return 1; }
+  static constexpr std::string_view getDescription() { return "Informations on solver time step"; }
+};
+
+struct SolverNextDt
+{
+  static constexpr int getMinLogLevel() { return 3; }
+  static constexpr std::string_view getDescription() { return "Informations on changing DT"; }
+};
+
+struct Dof
+{
+  static constexpr int getMinLogLevel() { return 2; }
+  static constexpr std::string_view getDescription() { return "The summary of declared fields and coupling"; }
+};
+
+struct PoromechanicsPhaseFraction
+{
+  static constexpr int getMinLogLevel() { return 1; }
+  static constexpr std::string_view getDescription() { return "Print phase volume fraction"; }
```

Comment:
same here - phase volume fraction but why it is called PoromechanicsPhaseFraction???

Reply:
It's related to Poromechanics but i coud rephrase with  "Print Poromechanics phase volume fraction"